resty/src/App.js

```diff
@@ -13,46 +14,47 @@ class App extends React.Component {
       count: '',
       resultsArray: [],
       headers: {},
-      history: {},
-      storage: {}
     }
   }
 
   updateState = (data) => {
-    this.setState({ resultsArray: data.results })
+    // console.log('get from fake api??', data);
+    this.setState({ resultsArray: data })
     this.setState({ count: data.count })
```

Comment:
When I try and look for data from a second api link, this causes the application to get stuck, it's coming back as an Undefined Rejection